### ISSUE TITLE
[BUG]: Probe failing only with Bed Level Visualizer plugin enabled

### ISSUE THREAD

**Issue by jordansilva** [jneilliii/OctoPrint-BedLevelVisualizer#626]
**Describe the bug**
I tried several times to update the bed level with bed level plugin visualizer enabled, by calling `G29 T` myself or calling the script via plugin using octoprint. Every time that I had the plugin enabled or tried via plugin I had a consistent Probe failure at the same point. `G29 T` worked every time I had Bed Visualizer Plugin disabled.

I never had any issue before using the plugin. The issue started after I updated all my plugins and OctoPrint to 1.9.2.

**Expected behavior**
To run the probe without failure.

**Debug Log**
[octoprint.log](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/files/12339262/octoprint.log)
[serial.log](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/files/12339275/serial.log)

```
M117 Homing all axes ; send message to printer display
G28 ; home all axes
M420 S0 ; Turning off bed leveling while probing, if firmware is set
; to restore after probing
M117 Creating the bed mesh levels ; send message to printer display
@BEDLEVELVISUALIZER ; tell the plugin to watch for reported mesh
G29 T ; run bilinear probing
M500 ; store mesh in EEPROM
M300 S440 P200 ; make calibration completed tones
M300 S660 P250
M300 S880 P300
M117 Bed mesh levels completed ; send message to printer display
```

**Screenshots**
![image](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/assets/1149526/ce610144-f716-4c1a-9ccc-1e62bdfa558d)

**Firmware and Version**
env.plugins.pi_support.model: Raspberry Pi 4 Model B Rev 1.2
env.plugins.pi_support.octopi_version: 0.18.0
env.python.pip: 20.3.3
env.python.version: 3.7.3
env.python.virtualenv: True
octoprint.version: 1.9.2
printer.firmware: Marlin 2.0.9.3 (Jan  5 2022 17:31:04)

**Comment by jneilliii** [comment]
Try it without the T on the G29 T line so that it's just `G29`. There's nothing in your serial.log that indicates a failure of any kind, it just never completes.

**Comment by jneilliii** [comment]
I found this article that covers a couple of possible reasons for probing failed error, I would focus on the options that don't require Marlin firmware changes as it seemed to have worked for you previously. One that stands out is where the screws are out of a wack so much that distance traveled is causing a failed probe point. 

https://www.3dprintgorilla.com/bltouch-probing-failed-marlin-firmware/

**Comment by jordansilva** [comment]
I have checked this article as well, and also tried `G29` but no luck. 
It was consistently failing at 6/49 (front right). 

I just reconnected every cable and raised up the side that was failing. 
Apparently it worked now. Checking the bed level I noticed my x axis is the one that was really messed up, so probably the issue was related to my X axis was crooked. 

Thanks for the assistance, nothing related with the plugin.
Thanks for the plugin :)
